lib/EnsEMBL/REST/Model/Registry.pm

```diff
@@ -361,7 +358,7 @@ sub _build_species_info {
     my $species_lc = ($species);
     push(@{$groups_lookup{$species_lc}}, $group);
 
-    if($group eq 'core' && $species !~ /Ancestral/) {
+    if($group eq 'core' && $species !~ /(Ancestral|DEFAULT)/) {
```

Comment:
Can I get a note here to know what DEFAULT is?

Reply:
Removed